### PR TITLE
net-p2p/airdcpp-webclient: Fix incorrect configuration parameter name

### PR DESCRIPTION
**Pull request by D-Code**
Setting the UMASK configuration parameter in the configuration file did not have any effect. That's because the UMASK configuration parameter name in the init script and conf file did not match.

Package-Manager: Portage-2.3.8, Repoman-2.3.1

### CODE REVIEW

**Comment by sbraz** [comment]
Wow, that's not pretty! Thanks for the fix!

**Comment by sbraz** [comment]
@D-Code I think you should revbump the ebuild too.

**Comment by monsieurp** [comment]
@sbraz can you formally accept those changes if you agree with them? thanks.

**Comment by sbraz** [comment]
@monsieurp Yes I agree with those changes. Maybe both commits should be rebased into one though?

File: net-p2p/airdcpp-webclient/airdcpp-webclient-2.1.0-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+RDEPEND="
+	app-arch/bzip2
+	dev-cpp/websocketpp
+	dev-libs/boost:=
+	dev-libs/geoip
+	dev-libs/leveldb:=
+	dev-libs/openssl:0=[-bindist]
+	net-libs/miniupnpc:=
+	sys-libs/zlib
```
Comment (SoapGentoo) [inline net-p2p/airdcpp-webclient/airdcpp-webclient-2.1.0-r1.ebuild]: needs :=

**Pull request closed without merging**